dali/image/jpeg_mem.cc

```diff
@@ -178,10 +178,14 @@ uint8* UncompressLow(const void* srcdata, FewerArgsForCompiler* argball) {
   JDIMENSION target_output_width = cinfo.output_width;
   JDIMENSION target_output_height = cinfo.output_height;
   JDIMENSION skipped_scanlines = 0;
+  int left_cor = 0;
+  int right_cor = 0;
 #if defined(LIBJPEG_TURBO_VERSION)
   if (flags.crop) {
     // Update target output height and width based on crop window.
     target_output_height = flags.crop_height;
+    left_cor = flags.crop_x == 0 ? 0 : 1;
+    right_cor = flags.crop_x + flags.crop_width == cinfo.output_width ? 0 : 1;
```

Comment:
Maybe a cast here to avoid signed/unsigned mismatch warning?

Reply:
Done